x-pack/plugins/security_solution/public/detection_engine/rule_management/components/rule_details/rule_definition_section.tsx

```diff
@@ -427,7 +428,10 @@ const HistoryWindowSize = ({ historyWindowStart }: HistoryWindowSizeProps) => {
 const prepareDefinitionSectionListItems = (
   rule: Partial<RuleResponse>,
   isInteractive: boolean,
-  savedQuery?: SavedQuery
+  savedQuery: SavedQuery | undefined,
+  {
+    isAlertSuppressionForThresholdRuleFeatureEnabled,
+  }: { isAlertSuppressionForThresholdRuleFeatureEnabled: boolean }
```

Comment:
Can we extract this type out and call this parameter `featureFlags`? I imagine we are going to have an evolving list of flags here, now with serverless

Reply:
The flag is mostly here to prevent feature leaking to customers in Serverless before doc page is ready, so it should not be long lived. Hence, the previous interface. But, you are right, in future there would be probably more flags in this area


So, I reused existing interface from

```
import type { ExperimentalFeatures } from '../../../../../common/experimental_features';
```